worlds/oot/__init__.py

```diff
@@ -1188,10 +1188,34 @@ def hint_type_players(hint_type: str) -> set:
 
     def fill_slot_data(self):
         self.collectible_flags_available.wait()
+        prizes = {}
+        bosses = {}
+        for location in [
+            'Links Pocket',
+            'Queen Gohma',
+            'King Dodongo',
+            'Barinade',
+            'Phantom Ganon',
+            'Volvagia',
+            'Morpha',
+            'Bongo Bongo',
+            'Twinrova',
+            # 'Ganon',
+        ]:
+            if not location == 'Links Pocket':
```

Comment:
```suggestion
            if location != "Links Pocket":
```

Also, these should all be using double quotes, not single quotes